ConsoleApp2/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace ConsoleApp2
+{
+
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine(Max(3, 5));
+            Console.WriteLine(Min(3, 5));
+            Console.WriteLine(TrySumIfOdd(4, 4, out bool y));
+            Console.WriteLine(TrySumIfOdd(4, 5, out bool z));
+            Console.WriteLine(Max(3, 7, 5));
+            Console.WriteLine(Max(3, 7, 12, 5));
+            Console.WriteLine(Repeat("Dima~", 7));
+        }
+        static double Max(double x, double y)
+        {
+            return Math.Max(x, y);
+        }
+        static double Min(double x, double y)
+        {
+            return Math.Min(x, y);
+        }
+        static bool TrySumIfOdd(int x, int y, out bool isodd)
```

Comment:
method in homework should multiply 2 numbers if at least one can be divided by 3